src/Components/Parameters/AppChangelog.vue

```diff
@@ -0,0 +1,80 @@
+<!--
+  - @copyright Copyright (c) 2024 Ferdinand Thiessen <opensource@fthiessen.de>
+  -
+  - @author Ferdinand Thiessen <opensource@fthiessen.de>
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<NcButton v-if="canShowChangelog" class="app-changelog-button" @click="handleClick">
+		{{ t('notifications', 'See what\'s new for {app}', { app: name }) }}
```

Comment:
Don't cheat this in here. It should be part of the message itself.

Also actions exist for buttons, as explained in the server pr

Reply:
The idea was to make this an object (clients can fetch using OCS) so that clients do not need to move the user to a web browser.  If they do not support it I expected them to use the already parsed message.